cmd/list/accessrequests/cmd.go

```diff
@@ -0,0 +1,116 @@
+package accessrequests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/accesstransparency/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use   = "access-request"
+	short = "List access requests"
+	long  = "List access requests in Pending or Approved status. " +
+		"If '--cluster' flag is used, list all access requests in any status for the specified cluster."
+	example = `  # List all access requests for cluster 'foo'
+  rosa list access-request --cluster foo
+  `
+)
+
+func NewListAccessRequestsCommand() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: []string{"accessrequest", "accessrequests", "access-requests"},
+		Short:   short,
+		Long:    long,
+		Example: example,
+		Run:     rosa.DefaultRunner(rosa.RuntimeWithOCM(), ListAccessRequestsRunner()),
+		Args:    cobra.NoArgs,
+	}
+
+	output.AddFlag(cmd)
+	ocm.AddOptionalClusterFlag(cmd)
+	return cmd
+}
+
+func ListAccessRequestsRunner() rosa.CommandRunner {
+	return func(_ context.Context, r *rosa.Runtime, cmd *cobra.Command, _ []string) error {
+		clusterId := ""
+		if cmd.Flags().Changed("cluster") {
+			cluster, err := r.OCMClient.GetCluster(r.GetClusterKey(), r.Creator)
+			if err != nil {
+				return err
+			}
+			clusterId = cluster.ID()
+		}
+		accessRequests, err := r.OCMClient.ListAccessRequest(clusterId)
+		if err != nil {
+			return err
+		}
+		if output.HasFlag() {
+			output.Print(accessRequests)
+		} else {
+			if len(accessRequests) == 0 {
+				if clusterId == "" {
+					r.Reporter.Infof("There are no Access Requests in Pending or Approved status.")
+				} else {
+					r.Reporter.Infof("There are no Access Requests for cluster '%s'.", r.ClusterKey)
+				}
+				return nil
+			}
+			writer := tabwriter.NewWriter(os.Stdout, 0, 4, 2, ' ', 0)
+			output, hasPending, pendingId := getAccessRequestsOutput(clusterId, accessRequests)
+			fmt.Fprint(writer, output)
+			if err := writer.Flush(); err != nil {
+				return err
+			}
+			if hasPending {
```

Comment:
@marcolan018 Are you OK with this message if the customer has multiple, pending Access Requests?

Reply:
@robpblake 
that's current implementation, if there is Pending Access Request
if user specify `--cluster`, then log out something like 
`rosa create decision --access-request 2nFVxq5ki0ktsRtmhrSyGf98TVB  --decision Approved`
if user does not specify `--cluster` flag, log out
`rosa create decision --access-request <ID>  --decision Approved`

do you have concern on this? And any recommendation about the output?